src/character.cpp

```diff
@@ -2306,3 +2282,63 @@ std::string Character::extended_description() const
 
     return replace_colors( ss.str() );
 }
+
+template <typename Getter>
+float Character::calc_mutation_value( Getter getter ) const
```

Comment:
I intend to replace it with a cache later on. Went with getter for implementation simplicity rather than usage simplicity.